xml/System.Windows/ThemeMode.xml

```diff
@@ -93,9 +101,9 @@
         <Parameter Name="obj" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="obj">ThemeMode object to compare with.</param>
+        <summary>Checks whether the object is equal to another ThemeMode object.</summary>
+        <returns>Returns <see langword="true" /> when the ThemeMode objects are equal, and <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" /> when the ThemeMode objects are equal, and <see langword="false" /> otherwise.</returns>
```